packages/webview_flutter/webview_flutter_wkwebview/lib/src/webkit_webview_controller.dart

```diff
@@ -326,6 +326,13 @@ class WebKitWebViewController extends PlatformWebViewController {
   Future<void> addJavaScriptChannel(
     JavaScriptChannelParams javaScriptChannelParams,
   ) {
+    final String channelName = javaScriptChannelParams.name;
```

Comment:
Is somewhere already checking this isn't an empty string?
> This parameter must be unique within the user content controller and must not be an empty string.

https://developer.apple.com/documentation/webkit/wkusercontentcontroller/1537172-addscriptmessagehandler

Reply:
Yea it checks it here with an assert: https://github.com/flutter/packages/blob/main/packages/webview_flutter/webview_flutter_wkwebview/lib/src/webkit_webview_controller.dart#L695